src/pytorch_lightning/trainer/configuration_validator.py

```diff
@@ -158,6 +159,10 @@ def __verify_dp_batch_transfer_support(trainer: "pl.Trainer", model: "pl.Lightni
             is_overridden(hook, datahook_selector.model) or is_overridden(hook, datahook_selector.datamodule)
         ):
             raise MisconfigurationException(f"Overriding `{hook}` is not supported in DP mode.")
+        if isinstance(trainer.accelerator, IPUAccelerator) and (
+            is_overridden(hook, datahook_selector.model) or is_overridden(hook, datahook_selector.datamodule)
```

Comment:
You reused this method, but it's meant as a DP warning.

I suggest you change the name to something like `__verify_batch_transfer_support` and move the TODO comment right beside the DP check

Reply:
done!